SpaceShooter.py

```diff
@@ -1,36 +1,44 @@
 from time import time as timer
+
+import pygame
 from pygame import *
-from random import randint
+from random import randint, choice
+pygame.init()
 
 clock = time.Clock()
 FPS = 60
 
 reload = False
 
-
-
 lost_monsters = 0
 killed_monsters = 0
 lives = 3
 color_g = (0, 255, 0)
 color_y = (255, 255, 0)
 color_r = (255, 0, 0)
-color_w=(255,255,255)
+color_w = (255, 255, 255)
 fired = 0
 
-window = display.set_mode((700, 500))
+info=display.Info()
+w=info.current_w
+h=info.current_h
+#w=430
+#h=932
+
+
+window = display.set_mode((w,h))
 display.set_caption("Шутер")
-background = transform.scale(image.load("galaxy.jpg"), (700, 500))
+background = transform.scale(image.load("galaxy.jpg"), (w,h))

```

Comment:
I think that there need using aspect ratio for mobile phones and computers
need add new validete for it